site/docs/v1/tech/tutorials/integrate-flutter.md

```diff
@@ -55,6 +55,20 @@ Doctor summary (to see all details, run flutter doctor -v):
 
 {% include docs/integration/_add-user.md language=page.language %}
 
+## Set up a Public URL for Fusionauth
+
+Your FusionAuth instance is now running on a different machine (your computer) than the mobile app will run (either a real device or an emulator), which means that it won't be able to access `localhost`.
+
+If the device (either real or emulator) and your computer are connected to the same network, you can use the local IP Address for your machine (e.g. `192.168.15.2`). Here are a few articles to help your find your IP Address depending on the Operational System you are running: 
+
+- [Linux](https://linuxize.com/post/how-to-find-ip-address-linux/#find-your-public-ip-address)
+- [Mac](https://support.apple.com/guide/mac-help/find-your-computers-name-and-network-address-mchlp1177/mac)
+- [Windows](https://support.microsoft.com/en-us/windows/find-your-ip-address-in-windows-f21a9bbc-c582-55cd-35e0-73431160a1b9)
+
+If they are not connected to the same network or if you have something that blocks connections (like a Firewall), you can use [ngrok](https://ngrok.com/docs/getting-started/) to expose your local FusionAuth instance running at port `9011` to the Internet.
```

Comment:
It would make sense to link to https://fusionauth.io/docs/v1/tech/developer-guide/exposing-instance instead of the ngrok website.

Reply:
Yep, that's way better :) Done! Thanks